src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/activities/ActivitiesMigrationIntTest.kt

```diff
@@ -79,7 +80,7 @@ class ActivitiesMigrationIntTest : SqsIntegrationTestBase() {
       migrationHistoryRepository.deleteAll()
     }
 
-    private fun WebTestClient.performMigration(body: String = """{ "prisonId": "BXI" }""") = post().uri("/migrate/activities")
+    private fun WebTestClient.performMigration(body: String = """{ "prisonId": "BXI", "activityStartDate": "$tomorrow" }""") = post().uri("/migrate/activities")
```

Comment:
Changed all tests to have a start date - nowadays there is always a start date in the filter when migrating.